sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -87,3 +66,27 @@
 - Support for Base64 Encoding/Decoding.
 - Support for MD5, CRC64.
 - Support for Shared Key Credential.
+## 12.0.0-beta.6 (2020-01-14)
+
+### New Features
+
+- Added new type `ContentHash`.
+- Added definition of `Metadata`.
+- Support setting account SAS permission with a raw string.
+
+### Breaking Changes
+
+- Renamed `SharedKeyCredential` to `StorageSharedKeyCredential`.
```

Comment:
Interesting. Either something in the CL format is broken or we need to debug why the update script regex is making these changes.

cc @danieljurek, @chidozieononiwu

Reply:
OK, I think I know what happend. The beta.6 release date is wrong/out-of-order. It is saying `2020-01-14`, when it should be `2021-01-14`.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-cpp/pull/1970